bulkExport/export-script.py

```diff
@@ -0,0 +1,105 @@
+"""
```

Comment:
Actually, can you move this file to a folder dedicated to glue scripts? Maybe `bulkExport/glueScripts/export-script.py`

The thing is that pretty much all upload-to-s3-on-deployment upload the contents of a **folder** to s3

Reply:
yep, doing that now